setup/lang/en/default.inc.php

```diff
@@ -213,6 +213,7 @@
 $_lang['toggle'] = 'Toggle';
 $_lang['toggle_success'] = 'Toggle Success Messages';
 $_lang['toggle_warnings'] = 'Toggle Warnings';
+$_lang['upgrade_version_unsupported'] = 'Upgrading from MODX [[+version]] is not supported by this release. You will need to upgrade to MODX 2.6 or later before upgrading to this release.';
```

Comment:
Why 2.6 and not 2.7?

Reply:
Probably could have supported older, but didn't want to deal with it. There were no upgrade scripts for 2.6, so no reason not to support 2.6+.